### PR TITLE
External content

### PR DESCRIPTION
**Pull request by rajeevsathish**

### CODE REVIEW

File: src/app/resourcebundles/data/messages/en.properties [hunk 1]
```diff
@@ -290,4 +290,5 @@ m0029=No organization selected!
 m0030=Addresses should not be of same type, please change any one.
 m0031=You already have a current job, please mark one job as current job
 m0032=Max upload size 4MB
-m0033=(Format: .pdf, .png, .jpg. File Limit: 1MB)
\ No newline at end of file
+m0033=(Format: .pdf, .png, .jpg. File Limit: 1MB)
+m0034=As the content is external source, so will be openend in a new tab
```
Comment (bvinayakumar) [inline src/app/resourcebundles/data/messages/en.properties]: Typo: Change openend to opened.

File: src/app/server.js [hunk 2]
```diff
@@ -32,14 +32,15 @@ const reqDataLimitOfContentEditor = '50mb'
...
+const externalWhitelistedDomains = envHelper.SUNBIRD_EXTERNAL_CONTENT_WHITELISTED_DOMAINS
 const defaultTenant = envHelper.DEFAUULT_TENANT
 const portal = this
 const Telemetry = require('sb_telemetry_util')
 const telemetry = new Telemetry()
 const telemtryEventConfig = JSON.parse(fs.readFileSync(path.join(__dirname, 'helpers/telemetryEventConfig.json')))
 const producerId = process.env.sunbird_environment + '.' + process.env.sunbird_instance + '.portal'
 let cassandraCP = envHelper.PORTAL_CASSANDRA_URLS
-const contentServiceLocalUrl = envHelper.content_Service_Local_BaseUrl
+// const contentServiceLocalUrl = envHelper.content_Service_Local_BaseUrl
```
Comment (bvinayakumar) [inline src/app/server.js]: Remove unused variable.

File: src/app/resourcebundles/data/messages/en.properties (hunk 1 above)
Comment (rajeevsathish) [reply src/app/resourcebundles/data/messages/en.properties]: Changes are done

File: src/app/server.js (hunk 2 above)
Comment (rajeevsathish) [reply src/app/server.js]: Changes are done this variable was Added by Anoop I commented it as the hook was not allowing me to commit as it was not used at all

**Comment by bvinayakumar** [comment]
Another pull request is raised so closing this one.

**Pull request closed without merging**